### ISSUE TITLE
Specify which panes to diff

### ISSUE THREAD

**Issue by johnsgill3** [mupchrch/split-diff#42]
A lot of times I'll have 2, 3 panes open (split down, split left, etc). It would be nice if I could specify which two panes I want to diff. Say I have 
` | pane 1 | pane 2 | pane 3 | `
Currently split diff only allows diff between panes 1 & 2. But say I want to diff 2 & 3. I have to move the file from 3 over to 1 just to diff.

Additionally say I have 
```
|      pane 1     |
| pane 2 | pane 3 |
```
split diff only allows the ability to diff panes 1 & 2 still, but I want to be able to diff 2 & 3.

**Comment by mupchrch** [comment]
Ah, good suggestion! So if there are more than 2 panes open, you would expect to have to click the panes which you want to diff. But if there are 2 or less, then split-diff would behave as is. I think that makes a lot of sense.

**Comment by johnsgill3** [comment]
Yeah I'm not exactly sure on how the behavior would work to specify the panes. Clicking on the two might not be a bad idea.

Also are there any plans to support more than two pane diffs? I know that if you use something like vimdiff it supports N 'panes'/files that can be diffed simultaneously.

**Comment by DallasO** [comment]
If you go with some sort of fancy popup overlay, I feel like this would be a good stand-alone project as there are many projects that could use this feature.

**Comment by mupchrch** [comment]
@johnsgill3 I've never actually used a multi diffing tool. I'll have to investigate and see if it makes sense / is possible.

@DallasO I'm not sure what you mean by "many projects that could use this feature". I would like to make a little modal window or popup at some point though. I think this package would make a whole lot more sense with a UI.

**Comment by mupchrch** [comment]
Is this still a pain point now that you can right click a tab and choose "Diff with Active File"?

**Comment by rainboxx** [comment]
One thing that could also fall into this issue is diffing the active file against itself. Use case: I have several panes and want to see changes to the current file only, not comparing it with other panes.
